src/plugins/paginate/paginate.js

```diff
@@ -234,7 +274,7 @@ const componentName = "wb-paginate",
 	};
 
 // When a filter is updated
```

Comment:
That was my miss. Could you please update the comment to the following:

```suggestion
// When a page button is clicked
```

Reply:
Fixed in commit "Made recommended changes".